paasta_tools/paasta_cluster_boost.py

```diff
@@ -26,6 +26,11 @@
 def parse_args():
     """Parses the command line arguments passed to this script"""
     parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
Do we need this to be an arg at all?
This command is running on master, just lookup the cluster you are a member of. We'll never be calling this command from some remote place.

Reply:
Yea, good call.  I'll take this out.